torch_disc/torch_disc/BUILD

```diff
@@ -1,16 +1,29 @@
 load("@pybind11_bazel//:build_defs.bzl", "pybind_extension", "pybind_library")
 
-pybind_library (
+cc_library (
   name = "torch_disc_backend",
-  srcs = ["csrc/backend_impl.cpp", "csrc/disc_jit.cpp"],
-  hdrs = ["csrc/backend_impl.h", "csrc/disc_jit.h"],
+  srcs = [
+    "csrc/backend_impl.cpp",
```

Comment:
Nit: Shall we create some packages such as `csrc/disc/frontend` and `csrc/disc/backend/`. And then move these source files to their suitable locations? I think that could help us to make codes more modulable comprehensible?

Reply:
Good idea, and will do that.

Reply:
done. I refactor the code structures:
- `CompileToDiscExecutable` is an entry function that takes a Torch graph to cluster and register Disc class, this function outputs an `Executable` instance.
- `Executable` executes the Graph with input values and outputs Tensors as the register outputs.